packages/p2p-v2/src/components/FormProgress/FormProgress.scss

```
@@ -0,0 +1,74 @@
+.p2p-v2-form-progress {
+    width: 100%;
+    position: relative;
+
+    &__header {
+        align-items: center;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        margin-bottom: 3.2rem;
+    }
+    &__step {
+        align-items: center;
+        display: flex;
+        flex-direction: column;
+        height: 6rem;
+        justify-content: space-around;
+        width: 160px;
```

Should this be `rem` @nada-deriv? 🤔 
```suggestion
        width: 160px;
```